dashboard/templates/control_panel.html

```diff
@@ -0,0 +1,40 @@
+{% extends "base.html" %}
+{% block title %}Control Panel{% endblock %}
+{% block content %}
+    <div class="mb-2 text-center">
+        <h1>Control panel</h1>
+    </div>
+    <div class="container">
+        <div class="row mt-5 mb-2">
+            <div class="col-md-12 col-lg-6 offset-lg-3">
+                <div class="table-responsive">
+                    <table class="table">
+                        <thead class="thead-dark">
+                            <tr>
+                                <th scope="col">Pipeline</th>
+                                <th scope="col">Branch</th>
+                                <th scope="col">Action</th>
+                            </tr>
+                        </thead>
+                        <tbody>
+                        {% for pipelines in start_pipelines %}
+                            {% for pipeline, branches in pipelines.items() %}
+                                {% for branch in branches %}
+                                    <tr>
+                                        <td> {{ pipeline }}</td>
+                                        <td> {{ branch }}</td>
+                                        <td>
+                                            <a class="btn btn-success" role="button" href="{{ url_for('build_enqueue.start_buildset', pipeline=pipeline, branch=branch) }}">START</a>
```

Comment:
We should check against running buildsets and show only start/stop when buildset is not or running.